lib/led/rgb.js

```diff
@@ -275,22 +281,21 @@ class RGB {
         blue: 255
       };
 
-      state.prev = null;
-
       this.update(colors);
+    } else if (state.isRunning) {
```

Comment:
Added this else-if branch for case when `on()` is called while RGB is pulsing so that RGB displays full RGB values and not 'tween' values of `pulse` method.